apps/agreement/contracts/test/mocks/helpers/TimeHelpersMock.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.4.24;
```

Comment:
Quick thing; does the version exported by `contract-test-helpers` not work?

Should we update that repo with this version?

Reply:
It does work, the problem is that when working with multiple contracts, in this case the agreement and disputable apps, you need to mock each of them individually. Here I modeled a clock that is shared among all these, so you only need to update one thing, and you can make sure all these instances have the same timestamp :)

So yes, I'd love to update the `contract-test-helpers` version :)

Reply:
Right, I was just thinking about how to do this (e.g. if you were building an app and wanted to mock time in other contracts as well).

Let's move it over!